cluster/addons/cluster-loadbalancing/glbc/glbc-controller.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: l7-lb-controller
+  name: l7-lb-controller-v0.5.2
```

Comment:
No tag for image here? or the name with version information will translate to image+tag?